crypto/src/main/java/net/corda/v5/crypto/CordaOID.java

```diff
@@ -0,0 +1,34 @@
+package net.corda.v5.crypto;
+
+/**
+ * OIDs used for the Corda platform. All entries MUST be defined in this file only and they MUST NOT be removed.
+ * If an OID is incorrectly assigned, it should be marked deprecated and NEVER be reused again.
+ */
+
+public final class CordaOID {
```

Comment:
Add a private no-argument constructor so that no-one can instantiate `CordaOID`.

Reply:
why would you want to? `CordaOID` is just a bunch of string constants, there's no logic there and no need I can see to instantiate anything.

Reply:
Let's just do it. No downside to it and if it's consistent with what we do elsewhere, that's a good enough reason.